string entry_id
---
bool success
float64 x_center
float64 y_center